CMakeLists.txt

```diff
@@ -3,7 +3,24 @@ project(openGA CXX C)
 set(CMAKE_CXX_STANDARD 11)
 set(THREADS_PREFER_PTHREAD_FLAG ON)
 find_package(Threads REQUIRED)
-find_package(Boost COMPONENTS system filesystem thread chrono REQUIRED)
+# Eigen
+include_directories("/usr/include/eigen3" )
+#TdLib
+set(TdLib_DIR "/home/wang/CLionProjects/TdLibrary/install/lib/CMake/TdLib/")
+find_package(TdLib REQUIRED)
+include_directories(${TdLib_INCLUDE_DIRS})
+MESSAGE(STATUS "TdLib_INCLUDE_DIRS = " ${TdLib_INCLUDE_DIRS})
+FIND_PACKAGE( OpenMP REQUIRED)
+if(OPENMP_FOUND)
+    message("OPENMP FOUND")
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${OpenMP_C_FLAGS}")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
+endif()
```

Comment:
Is there any particular reason to prefer `OpenMP` over `std::thread`?

Reply:
The OpenMP is for two GA running parallel. It is easier than std::thread. This modification is just for me. I don't request a pull request for this.